{"data":[{"url":"https://api.github.com/repos/renpy/renpy/issues/1117","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1117/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1117/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1117/events","html_url":"https://github.com/renpy/renpy/issues/1117","id":202536379,"number":1117,"title":"Graphical trash on the next lines using outlines at say_dialogue","user":{"login":"Ria-kon","id":24704640,"avatar_url":"https://avatars2.githubusercontent.com/u/24704640?v=4","gravatar_id":"","url":"https://api.github.com/users/Ria-kon","html_url":"https://github.com/Ria-kon","followers_url":"https://api.github.com/users/Ria-kon/followers","following_url":"https://api.github.com/users/Ria-kon/following{/other_user}","gists_url":"https://api.github.com/users/Ria-kon/gists{/gist_id}","starred_url":"https://api.github.com/users/Ria-kon/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/Ria-kon/subscriptions","organizations_url":"https://api.github.com/users/Ria-kon/orgs","repos_url":"https://api.github.com/users/Ria-kon/repos","events_url":"https://api.github.com/users/Ria-kon/events{/privacy}","received_events_url":"https://api.github.com/users/Ria-kon/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":2,"created_at":"2017-01-23T13:40:58Z","updated_at":"2017-01-24T02:28:55Z","closed_at":"2017-01-23T15:26:27Z","author_association":"CONTRIBUTOR","body":"Watch gif example\r\n![1](https://cloud.githubusercontent.com/assets/24704640/22206026/1e204440-e182-11e6-8a4f-4a29ff83681f.gif)\r\ngraphical trash appears only from this outlines code\r\n```\r\ninit -1 style say_dialogue:\r\n    xpos 220\r\n    xanchor gui.text_xalign\r\n    xsize 840\r\n    line_spacing 3\r\n    ypos gui.text_ypos\r\n    justify True\r\n    text_align gui.text_xalign\r\n    layout (\"subtitle\" if gui.text_xalign else \"tex\")\r\n    outlines [(2, gui.text_outline_color, 0, 1),(1.5, gui.text_outline_color, 0, 0)]\r\n```","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1117/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1116","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1116/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1116/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1116/events","html_url":"https://github.com/renpy/renpy/issues/1116","id":202318317,"number":1116,"title":"Bug or not?","user":{"login":"XelaPy","id":7989489,"avatar_url":"https://avatars2.githubusercontent.com/u/7989489?v=4","gravatar_id":"","url":"https://api.github.com/users/XelaPy","html_url":"https://github.com/XelaPy","followers_url":"https://api.github.com/users/XelaPy/followers","following_url":"https://api.github.com/users/XelaPy/following{/other_user}","gists_url":"https://api.github.com/users/XelaPy/gists{/gist_id}","starred_url":"https://api.github.com/users/XelaPy/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/XelaPy/subscriptions","organizations_url":"https://api.github.com/users/XelaPy/orgs","repos_url":"https://api.github.com/users/XelaPy/repos","events_url":"https://api.github.com/users/XelaPy/events{/privacy}","received_events_url":"https://api.github.com/users/XelaPy/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":3,"created_at":"2017-01-21T14:25:13Z","updated_at":"2017-02-06T06:46:26Z","closed_at":"2017-02-06T06:46:26Z","author_association":"CONTRIBUTOR","body":"Under this setup, the second text displayable gets stuck on Cat when the script makes it to Cow. I am not 100% sure if it's a bug or expected behavior as it does not occur with window set on \"auto\"... as usual original code is very long but it all comes down to this:\r\n\r\n```renpy\r\n# options.rpy\r\ndefine config.window = \"hide\"\r\n\r\n# screens.rpy\r\nscreen say(who, what):\r\n    style_prefix \"say\"\r\n\r\n    if who is not None:\r\n        vbox:\r\n            text who\r\n            fixed:\r\n                at Transform(yzoom=.6)\r\n                text who at Transform(alpha=.6) # Second text displayable\r\n                \r\n    # ... rest of the normal say screen...\r\n    \r\n# script.rpy\r\nlabel start:\r\n    \"Cat\" \"Meow!\"\r\n    \"Cow\" \"Mooo!\"","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1116/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1115","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1115/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1115/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1115/events","html_url":"https://github.com/renpy/renpy/issues/1115","id":202029144,"number":1115,"title":"How can i import and use python package(standrd and 3rd) in Renpy?","user":{"login":"decemer","id":6077595,"avatar_url":"https://avatars3.githubusercontent.com/u/6077595?v=4","gravatar_id":"","url":"https://api.github.com/users/decemer","html_url":"https://github.com/decemer","followers_url":"https://api.github.com/users/decemer/followers","following_url":"https://api.github.com/users/decemer/following{/other_user}","gists_url":"https://api.github.com/users/decemer/gists{/gist_id}","starred_url":"https://api.github.com/users/decemer/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/decemer/subscriptions","organizations_url":"https://api.github.com/users/decemer/orgs","repos_url":"https://api.github.com/users/decemer/repos","events_url":"https://api.github.com/users/decemer/events{/privacy}","received_events_url":"https://api.github.com/users/decemer/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":2,"created_at":"2017-01-20T02:05:46Z","updated_at":"2017-01-24T04:05:20Z","closed_at":"2017-01-20T05:18:03Z","author_association":"NONE","body":"How can i import and use python package(standrd and 3rd) in Renpy?\r\n---------------\r\nfor example\r\n`init python: import socket`\r\nhow to store in game file folder?\r\ncan it run in PC,IOS,Android?","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1115/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1114","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1114/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1114/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1114/events","html_url":"https://github.com/renpy/renpy/issues/1114","id":201737660,"number":1114,"title":"replace_text only functions if a custom_text_tag has been defined","user":{"login":"Doctus","id":535700,"avatar_url":"https://avatars0.githubusercontent.com/u/535700?v=4","gravatar_id":"","url":"https://api.github.com/users/Doctus","html_url":"https://github.com/Doctus","followers_url":"https://api.github.com/users/Doctus/followers","following_url":"https://api.github.com/users/Doctus/following{/other_user}","gists_url":"https://api.github.com/users/Doctus/gists{/gist_id}","starred_url":"https://api.github.com/users/Doctus/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/Doctus/subscriptions","organizations_url":"https://api.github.com/users/Doctus/orgs","repos_url":"https://api.github.com/users/Doctus/repos","events_url":"https://api.github.com/users/Doctus/events{/privacy}","received_events_url":"https://api.github.com/users/Doctus/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":0,"created_at":"2017-01-19T01:08:26Z","updated_at":"2017-01-20T03:05:56Z","closed_at":"2017-01-20T03:05:56Z","author_association":"NONE","body":"In text/text.py, `renpy.config.replace_text` is only applied within a function that's only called `if renpy.config.custom_text_tags`. Therefore, if you haven't set any custom text tags at all, the empty dict evaluates False and config.replace_text has no effect.\r\n\r\nThis is probably responsible for past reports of replace_text not working, e.g. https://lemmasoft.renai.us/forums/viewtopic.php?f=8&t=38198","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1114/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1113","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1113/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1113/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1113/events","html_url":"https://github.com/renpy/renpy/issues/1113","id":200921175,"number":1113,"title":"Scaling window make text in Help goes out-of-frame (new GUI)","user":{"login":"Ria-kon","id":24704640,"avatar_url":"https://avatars2.githubusercontent.com/u/24704640?v=4","gravatar_id":"","url":"https://api.github.com/users/Ria-kon","html_url":"https://github.com/Ria-kon","followers_url":"https://api.github.com/users/Ria-kon/followers","following_url":"https://api.github.com/users/Ria-kon/following{/other_user}","gists_url":"https://api.github.com/users/Ria-kon/gists{/gist_id}","starred_url":"https://api.github.com/users/Ria-kon/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/Ria-kon/subscriptions","organizations_url":"https://api.github.com/users/Ria-kon/orgs","repos_url":"https://api.github.com/users/Ria-kon/repos","events_url":"https://api.github.com/users/Ria-kon/events{/privacy}","received_events_url":"https://api.github.com/users/Ria-kon/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2017-01-16T03:01:09Z","updated_at":"2017-01-16T03:17:11Z","closed_at":"2017-01-16T03:16:50Z","author_association":"CONTRIBUTOR","body":"1280x720 (TAB) ![screenshot0001](https://cloud.githubusercontent.com/assets/24704640/21969489/6536fcd8-dba7-11e6-93fa-f33f521136d3.png)\r\n1251x703 (ideal) ![screenshot0004](https://cloud.githubusercontent.com/assets/24704640/21969501/7f3b6c0e-dba7-11e6-88d6-e74c007c0ab3.png)\r\n1176x662 (CTRL, H) ![screenshot0002](https://cloud.githubusercontent.com/assets/24704640/21969538/b733b10c-dba7-11e6-98fd-88cc1b57c777.png)\r\n960x540 (TAB, H, g all over the text) ![screenshot0003](https://cloud.githubusercontent.com/assets/24704640/21969566/ebf7905c-dba7-11e6-9368-4b910e4d8f09.png)","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1113/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1112","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1112/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1112/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1112/events","html_url":"https://github.com/renpy/renpy/issues/1112","id":200784733,"number":1112,"title":"Possible unicode issue?","user":{"login":"renpytom","id":379804,"avatar_url":"https://avatars1.githubusercontent.com/u/379804?v=4","gravatar_id":"","url":"https://api.github.com/users/renpytom","html_url":"https://github.com/renpytom","followers_url":"https://api.github.com/users/renpytom/followers","following_url":"https://api.github.com/users/renpytom/following{/other_user}","gists_url":"https://api.github.com/users/renpytom/gists{/gist_id}","starred_url":"https://api.github.com/users/renpytom/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/renpytom/subscriptions","organizations_url":"https://api.github.com/users/renpytom/orgs","repos_url":"https://api.github.com/users/renpytom/repos","events_url":"https://api.github.com/users/renpytom/events{/privacy}","received_events_url":"https://api.github.com/users/renpytom/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2017-01-14T06:17:51Z","updated_at":"2017-01-17T05:46:00Z","closed_at":"2017-01-17T05:46:00Z","author_association":"OWNER","body":"https://lemmasoft.renai.us/forums/viewtopic.php?f=8&t=42028","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1112/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1111","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1111/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1111/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1111/events","html_url":"https://github.com/renpy/renpy/pull/1111","id":199947980,"number":1111,"title":"Russian translation by Project Gardares","user":{"login":"Ria-kon","id":24704640,"avatar_url":"https://avatars2.githubusercontent.com/u/24704640?v=4","gravatar_id":"","url":"https://api.github.com/users/Ria-kon","html_url":"https://github.com/Ria-kon","followers_url":"https://api.github.com/users/Ria-kon/followers","following_url":"https://api.github.com/users/Ria-kon/following{/other_user}","gists_url":"https://api.github.com/users/Ria-kon/gists{/gist_id}","starred_url":"https://api.github.com/users/Ria-kon/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/Ria-kon/subscriptions","organizations_url":"https://api.github.com/users/Ria-kon/orgs","repos_url":"https://api.github.com/users/Ria-kon/repos","events_url":"https://api.github.com/users/Ria-kon/events{/privacy}","received_events_url":"https://api.github.com/users/Ria-kon/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":4,"created_at":"2017-01-10T22:12:26Z","updated_at":"2017-01-16T03:27:26Z","closed_at":"2017-01-16T03:27:03Z","author_association":"CONTRIBUTOR","pull_request":{"url":"https://api.github.com/repos/renpy/renpy/pulls/1111","html_url":"https://github.com/renpy/renpy/pull/1111","diff_url":"https://github.com/renpy/renpy/pull/1111.diff","patch_url":"https://github.com/renpy/renpy/pull/1111.patch"},"body":"Done:\r\n1. Full russian translation for new GUI and for the launcher\r\n2. Text edited and optimized for russian language norms\r\n3. All files yoficated (https://en.wikipedia.org/wiki/Yoficator)\r\nNotes:\r\n4. Not all translated lines tested. May be some out-of-frames (in error messages for example)\r\n5. 13 issues for non-100%-sure lines\r\n6. Renpy is female now. =D (https://en.wikipedia.org/wiki/Grammatical_gender)","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1111/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1110","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1110/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1110/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1110/events","html_url":"https://github.com/renpy/renpy/pull/1110","id":199714390,"number":1110,"title":"Include widget stack in traceback when relevant","user":{"login":"SpikyCaterpillar","id":4422093,"avatar_url":"https://avatars0.githubusercontent.com/u/4422093?v=4","gravatar_id":"","url":"https://api.github.com/users/SpikyCaterpillar","html_url":"https://github.com/SpikyCaterpillar","followers_url":"https://api.github.com/users/SpikyCaterpillar/followers","following_url":"https://api.github.com/users/SpikyCaterpillar/following{/other_user}","gists_url":"https://api.github.com/users/SpikyCaterpillar/gists{/gist_id}","starred_url":"https://api.github.com/users/SpikyCaterpillar/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/SpikyCaterpillar/subscriptions","organizations_url":"https://api.github.com/users/SpikyCaterpillar/orgs","repos_url":"https://api.github.com/users/SpikyCaterpillar/repos","events_url":"https://api.github.com/users/SpikyCaterpillar/events{/privacy}","received_events_url":"https://api.github.com/users/SpikyCaterpillar/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2017-01-10T02:07:49Z","updated_at":"2017-01-10T05:07:53Z","closed_at":"2017-01-10T05:06:31Z","author_association":"CONTRIBUTOR","pull_request":{"url":"https://api.github.com/repos/renpy/renpy/pulls/1110","html_url":"https://github.com/renpy/renpy/pull/1110","diff_url":"https://github.com/renpy/renpy/pull/1110.diff","patch_url":"https://github.com/renpy/renpy/pull/1110.patch"},"body":"When a game crashes because of an unclosed layer/widget stack, includes the widget stack in the traceback.","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1110/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1109","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1109/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1109/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1109/events","html_url":"https://github.com/renpy/renpy/issues/1109","id":199344014,"number":1109,"title":"Issue with ALSA when trying to run Ren'Py 6.99.11 or higher on Travis CI","user":{"login":"NumberA","id":24855455,"avatar_url":"https://avatars2.githubusercontent.com/u/24855455?v=4","gravatar_id":"","url":"https://api.github.com/users/NumberA","html_url":"https://github.com/NumberA","followers_url":"https://api.github.com/users/NumberA/followers","following_url":"https://api.github.com/users/NumberA/following{/other_user}","gists_url":"https://api.github.com/users/NumberA/gists{/gist_id}","starred_url":"https://api.github.com/users/NumberA/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/NumberA/subscriptions","organizations_url":"https://api.github.com/users/NumberA/orgs","repos_url":"https://api.github.com/users/NumberA/repos","events_url":"https://api.github.com/users/NumberA/events{/privacy}","received_events_url":"https://api.github.com/users/NumberA/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":6,"created_at":"2017-01-07T05:51:39Z","updated_at":"2017-01-12T02:04:12Z","closed_at":"2017-01-12T02:04:12Z","author_association":"NONE","body":"Hey! I'm trying to integrate a Ren'Py project here on GItHub with Travis CI as descibed in [this](http://fuckyeahrenpy.tumblr.com/post/115884786323/automatically-testing-a-renpy-game-using-travis) tutorial. Unfortunately, when I try to build it fails with \"Exception: ALSA: Couldn't open audio device: No such file or directory\".\r\n\r\nTo troubleshoot, I reached out to [ExcaliburZero](https://gist.github.com/ExcaliburZero), author of the original tutorial, and he suggested I try dropping back to an older version of Ren'Py (he suggested 6.99.3). I did so, and that fixed the ALSA error, but then I got a bunch of Expression Expected lint errors.\r\n\r\nI was able to repro these Expression Expected errors on an Ubuntu VM by running my project (available on GitHub [here](https://github.com/NumberA/ProjectGaymore)) through Lint from the 6.99.3 Ren'Py SDK. I ran Lint from command line using **./renpy.sh \"../ProjectGaymore/\" lint**. When I ran the same command from the 6.99.12 Ren'Py SDK it completed without errors (printing some statistics to the terminal and then closing).\r\n\r\nThe final stage of my investigation was to return to the Travis CI tests and increment the Ren'Py SDK version I was running them against until the error switched from lint errors to ALSA exceptions. That happened between 6.99.10 and 6.99.11.  My test logs are available [here](https://travis-ci.org/NumberA/ProjectGaymore/builds) if they're useful.\r\n\r\nFrom what I can tell, the Travis CI test environment is \tUbuntu 12.04 LTS Server Edition 64 bit ([source](https://docs.travis-ci.com/user/ci-environment/)). Is that an OS that we can expect Ren'Py to work in?\r\n\r\nPlease let me know if you need any more information or if there's anything I can do to help get to the bottom of this.","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1109/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1108","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1108/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1108/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1108/events","html_url":"https://github.com/renpy/renpy/pull/1108","id":198756472,"number":1108,"title":"toggle_afm","user":{"login":"vollschauer","id":5165216,"avatar_url":"https://avatars1.githubusercontent.com/u/5165216?v=4","gravatar_id":"","url":"https://api.github.com/users/vollschauer","html_url":"https://github.com/vollschauer","followers_url":"https://api.github.com/users/vollschauer/followers","following_url":"https://api.github.com/users/vollschauer/following{/other_user}","gists_url":"https://api.github.com/users/vollschauer/gists{/gist_id}","starred_url":"https://api.github.com/users/vollschauer/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/vollschauer/subscriptions","organizations_url":"https://api.github.com/users/vollschauer/orgs","repos_url":"https://api.github.com/users/vollschauer/repos","events_url":"https://api.github.com/users/vollschauer/events{/privacy}","received_events_url":"https://api.github.com/users/vollschauer/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":5,"created_at":"2017-01-04T16:55:45Z","updated_at":"2017-01-17T05:48:17Z","closed_at":"2017-01-17T05:48:10Z","author_association":"CONTRIBUTOR","pull_request":{"url":"https://api.github.com/repos/renpy/renpy/pulls/1108","html_url":"https://github.com/renpy/renpy/pull/1108","diff_url":"https://github.com/renpy/renpy/pull/1108.diff","patch_url":"https://github.com/renpy/renpy/pull/1108.patch"},"body":"Simply bind 'a' to toggle afm ... ","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1108/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1107","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1107/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1107/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1107/events","html_url":"https://github.com/renpy/renpy/issues/1107","id":198749873,"number":1107,"title":"Not working: define config.has_autosave = False","user":{"login":"xavi-mat","id":1102677,"avatar_url":"https://avatars2.githubusercontent.com/u/1102677?v=4","gravatar_id":"","url":"https://api.github.com/users/xavi-mat","html_url":"https://github.com/xavi-mat","followers_url":"https://api.github.com/users/xavi-mat/followers","following_url":"https://api.github.com/users/xavi-mat/following{/other_user}","gists_url":"https://api.github.com/users/xavi-mat/gists{/gist_id}","starred_url":"https://api.github.com/users/xavi-mat/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/xavi-mat/subscriptions","organizations_url":"https://api.github.com/users/xavi-mat/orgs","repos_url":"https://api.github.com/users/xavi-mat/repos","events_url":"https://api.github.com/users/xavi-mat/events{/privacy}","received_events_url":"https://api.github.com/users/xavi-mat/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":0,"created_at":"2017-01-04T16:29:48Z","updated_at":"2017-01-05T01:32:32Z","closed_at":"2017-01-05T01:32:32Z","author_association":"MEMBER","body":"Simply add the line:\r\n    `define config.has_autosave = False`\r\nto a new game in options.rpy.\r\nIt still autosaves (the load screen shows the saves and they load).","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1107/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1106","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1106/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1106/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1106/events","html_url":"https://github.com/renpy/renpy/pull/1106","id":198654960,"number":1106,"title":"Update about.rpy","user":{"login":"doyousketch2","id":3681804,"avatar_url":"https://avatars0.githubusercontent.com/u/3681804?v=4","gravatar_id":"","url":"https://api.github.com/users/doyousketch2","html_url":"https://github.com/doyousketch2","followers_url":"https://api.github.com/users/doyousketch2/followers","following_url":"https://api.github.com/users/doyousketch2/following{/other_user}","gists_url":"https://api.github.com/users/doyousketch2/gists{/gist_id}","starred_url":"https://api.github.com/users/doyousketch2/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/doyousketch2/subscriptions","organizations_url":"https://api.github.com/users/doyousketch2/orgs","repos_url":"https://api.github.com/users/doyousketch2/repos","events_url":"https://api.github.com/users/doyousketch2/events{/privacy}","received_events_url":"https://api.github.com/users/doyousketch2/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":10,"created_at":"2017-01-04T08:37:31Z","updated_at":"2017-01-07T05:27:16Z","closed_at":"2017-01-07T05:03:36Z","author_association":"NONE","pull_request":{"url":"https://api.github.com/repos/renpy/renpy/pulls/1106","html_url":"https://github.com/renpy/renpy/pull/1106","diff_url":"https://github.com/renpy/renpy/pull/1106.diff","patch_url":"https://github.com/renpy/renpy/pull/1106.patch"},"body":"Added Python version info to the About screen.\r\n\r\n![Image](https://pbs.twimg.com/media/C1UEbhaW8AAvJDF.png:orig)","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1106/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1105","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1105/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1105/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1105/events","html_url":"https://github.com/renpy/renpy/issues/1105","id":198360517,"number":1105,"title":"imagebutton 'focus_mask True' doesn't work at all on Android OS","user":{"login":"lukurenpy","id":24879066,"avatar_url":"https://avatars2.githubusercontent.com/u/24879066?v=4","gravatar_id":"","url":"https://api.github.com/users/lukurenpy","html_url":"https://github.com/lukurenpy","followers_url":"https://api.github.com/users/lukurenpy/followers","following_url":"https://api.github.com/users/lukurenpy/following{/other_user}","gists_url":"https://api.github.com/users/lukurenpy/gists{/gist_id}","starred_url":"https://api.github.com/users/lukurenpy/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/lukurenpy/subscriptions","organizations_url":"https://api.github.com/users/lukurenpy/orgs","repos_url":"https://api.github.com/users/lukurenpy/repos","events_url":"https://api.github.com/users/lukurenpy/events{/privacy}","received_events_url":"https://api.github.com/users/lukurenpy/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2017-01-02T17:04:32Z","updated_at":"2017-01-03T16:44:16Z","closed_at":"2017-01-03T16:44:16Z","author_association":"NONE","body":"Hello,\r\nI noticed that `focus_mask` doesn't work at all on Android OS. Anywhere I tap on the screen, it will select the last added imagebutton, this might be because of `focus_mask` not working.\r\nOn desktop `focus_mask True` works as intended, I can select by clicking any imagebutton.\r\n\r\nPossible related information:\r\nIn Ren'Py's documentation in Android section there is this: \r\n\r\n> Ren'Py can't handle transparency in buttons and imagemaps. (This is due to performance problems on some devices with the display modes needed to support deciding which pixels are transparent.)\r\n\r\nSo it might be related.\r\n\r\n\r\nDevice used for testing: OnePlus X (Snapdragon 801, Quad-core 2.3 GHz Krait 400 (ARM architecture), Adreno 330)\r\n\r\nIs there a workaround for this issue on Android?","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1105/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1104","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1104/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1104/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1104/events","html_url":"https://github.com/renpy/renpy/issues/1104","id":198278188,"number":1104,"title":"Exception(\"A translation for u'[version!q]' already exists.\",)","user":{"login":"sl1pkn07","id":462213,"avatar_url":"https://avatars2.githubusercontent.com/u/462213?v=4","gravatar_id":"","url":"https://api.github.com/users/sl1pkn07","html_url":"https://github.com/sl1pkn07","followers_url":"https://api.github.com/users/sl1pkn07/followers","following_url":"https://api.github.com/users/sl1pkn07/following{/other_user}","gists_url":"https://api.github.com/users/sl1pkn07/gists{/gist_id}","starred_url":"https://api.github.com/users/sl1pkn07/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/sl1pkn07/subscriptions","organizations_url":"https://api.github.com/users/sl1pkn07/orgs","repos_url":"https://api.github.com/users/sl1pkn07/repos","events_url":"https://api.github.com/users/sl1pkn07/events{/privacy}","received_events_url":"https://api.github.com/users/sl1pkn07/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":5,"created_at":"2017-01-01T19:16:57Z","updated_at":"2017-01-01T20:10:14Z","closed_at":"2017-01-01T20:08:03Z","author_association":"NONE","body":"```\r\nException(\"A translation for u'[version!q]' already exists.\",)\r\n\r\nFull traceback:\r\n  File \"game/tl/arabic/launcher.rpy\", line 5, in script\r\n    old \"[version!q]\"\r\n  File \"/opt/renpy/renpy/ast.py\", line 2068, in execute\r\n    renpy.translation.add_string_translation(self.language, self.old, self.new, newloc)\r\n  File \"/opt/renpy/renpy/translation/__init__.py\", line 404, in add_string_translation\r\n    stl.add(old, new, newloc)\r\n  File \"/opt/renpy/renpy/translation/__init__.py\", line 347, in add\r\n    raise Exception(\"A translation for %r already exists.\" % old)\r\nException: A translation for u'[version!q]' already exists.\r\n\r\nWhile running game code:\r\n  File \"game/tl/arabic/launcher.rpy\", line 5, in script\r\n    old \"[version!q]\"\r\nException: A translation for u'[version!q]' already exists.\r\nException(\"A translation for u'[version!q]' already exists.\",)\r\n\r\nFull traceback:\r\n  File \"/opt/renpy/renpy/bootstrap.py\", line 290, in bootstrap\r\n    renpy.main.main()\r\n  File \"/opt/renpy/renpy/main.py\", line 419, in main\r\n    game.context().run(node)\r\n  File \"game/tl/arabic/launcher.rpy\", line 5, in script\r\n    old \"[version!q]\"\r\n  File \"/opt/renpy/renpy/ast.py\", line 2068, in execute\r\n    renpy.translation.add_string_translation(self.language, self.old, self.new, newloc)\r\n  File \"/opt/renpy/renpy/translation/__init__.py\", line 404, in add_string_translation\r\n    stl.add(old, new, newloc)\r\n  File \"/opt/renpy/renpy/translation/__init__.py\", line 347, in add\r\n    raise Exception(\"A translation for %r already exists.\" % old)\r\nException: A translation for u'[version!q]' already exists.\r\n\r\nWhile running game code:\r\n  File \"game/tl/arabic/launcher.rpy\", line 5, in script\r\n    old \"[version!q]\"\r\nException: A translation for u'[version!q]' already exists.\r\n```","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1104/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1103","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1103/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1103/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1103/events","html_url":"https://github.com/renpy/renpy/pull/1103","id":197735146,"number":1103,"title":"Fix typos in new GUI docs","user":{"login":"morganwillcock","id":14958747,"avatar_url":"https://avatars1.githubusercontent.com/u/14958747?v=4","gravatar_id":"","url":"https://api.github.com/users/morganwillcock","html_url":"https://github.com/morganwillcock","followers_url":"https://api.github.com/users/morganwillcock/followers","following_url":"https://api.github.com/users/morganwillcock/following{/other_user}","gists_url":"https://api.github.com/users/morganwillcock/gists{/gist_id}","starred_url":"https://api.github.com/users/morganwillcock/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/morganwillcock/subscriptions","organizations_url":"https://api.github.com/users/morganwillcock/orgs","repos_url":"https://api.github.com/users/morganwillcock/repos","events_url":"https://api.github.com/users/morganwillcock/events{/privacy}","received_events_url":"https://api.github.com/users/morganwillcock/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":2,"created_at":"2016-12-27T17:38:55Z","updated_at":"2016-12-29T02:37:57Z","closed_at":"2016-12-29T02:37:38Z","author_association":"CONTRIBUTOR","pull_request":{"url":"https://api.github.com/repos/renpy/renpy/pulls/1103","html_url":"https://github.com/renpy/renpy/pull/1103","diff_url":"https://github.com/renpy/renpy/pull/1103.diff","patch_url":"https://github.com/renpy/renpy/pull/1103.patch"},"body":"","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1103/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1102","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1102/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1102/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1102/events","html_url":"https://github.com/renpy/renpy/pull/1102","id":197517450,"number":1102,"title":"Replace 'open()' with 'with open()'","user":{"login":"jsfehler","id":7795990,"avatar_url":"https://avatars1.githubusercontent.com/u/7795990?v=4","gravatar_id":"","url":"https://api.github.com/users/jsfehler","html_url":"https://github.com/jsfehler","followers_url":"https://api.github.com/users/jsfehler/followers","following_url":"https://api.github.com/users/jsfehler/following{/other_user}","gists_url":"https://api.github.com/users/jsfehler/gists{/gist_id}","starred_url":"https://api.github.com/users/jsfehler/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/jsfehler/subscriptions","organizations_url":"https://api.github.com/users/jsfehler/orgs","repos_url":"https://api.github.com/users/jsfehler/repos","events_url":"https://api.github.com/users/jsfehler/events{/privacy}","received_events_url":"https://api.github.com/users/jsfehler/received_events","type":"User","site_admin":false},"labels":[],"state":"open","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":5,"created_at":"2016-12-25T22:34:34Z","updated_at":"2017-01-04T15:42:13Z","closed_at":null,"author_association":"CONTRIBUTOR","pull_request":{"url":"https://api.github.com/repos/renpy/renpy/pulls/1102","html_url":"https://github.com/renpy/renpy/pull/1102","diff_url":"https://github.com/renpy/renpy/pull/1102.diff","patch_url":"https://github.com/renpy/renpy/pull/1102.patch"},"body":"I wish I could think of a commit message that didn't use the word \"with\" twice.","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1102/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1101","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1101/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1101/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1101/events","html_url":"https://github.com/renpy/renpy/pull/1101","id":197501428,"number":1101,"title":"Fix exceptions to be compatible with Python 3.","user":{"login":"kibertoad","id":1847934,"avatar_url":"https://avatars3.githubusercontent.com/u/1847934?v=4","gravatar_id":"","url":"https://api.github.com/users/kibertoad","html_url":"https://github.com/kibertoad","followers_url":"https://api.github.com/users/kibertoad/followers","following_url":"https://api.github.com/users/kibertoad/following{/other_user}","gists_url":"https://api.github.com/users/kibertoad/gists{/gist_id}","starred_url":"https://api.github.com/users/kibertoad/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/kibertoad/subscriptions","organizations_url":"https://api.github.com/users/kibertoad/orgs","repos_url":"https://api.github.com/users/kibertoad/repos","events_url":"https://api.github.com/users/kibertoad/events{/privacy}","received_events_url":"https://api.github.com/users/kibertoad/received_events","type":"User","site_admin":false},"labels":[],"state":"open","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":5,"created_at":"2016-12-25T13:37:39Z","updated_at":"2016-12-25T14:52:40Z","closed_at":null,"author_association":"NONE","pull_request":{"url":"https://api.github.com/repos/renpy/renpy/pulls/1101","html_url":"https://github.com/renpy/renpy/pull/1101","diff_url":"https://github.com/renpy/renpy/pull/1101.diff","patch_url":"https://github.com/renpy/renpy/pull/1101.patch"},"body":"refs #1098","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1101/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1100","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1100/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1100/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1100/events","html_url":"https://github.com/renpy/renpy/pull/1100","id":197472992,"number":1100,"title":"Python3 port WIP","user":{"login":"caryoscelus","id":1758893,"avatar_url":"https://avatars3.githubusercontent.com/u/1758893?v=4","gravatar_id":"","url":"https://api.github.com/users/caryoscelus","html_url":"https://github.com/caryoscelus","followers_url":"https://api.github.com/users/caryoscelus/followers","following_url":"https://api.github.com/users/caryoscelus/following{/other_user}","gists_url":"https://api.github.com/users/caryoscelus/gists{/gist_id}","starred_url":"https://api.github.com/users/caryoscelus/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/caryoscelus/subscriptions","organizations_url":"https://api.github.com/users/caryoscelus/orgs","repos_url":"https://api.github.com/users/caryoscelus/repos","events_url":"https://api.github.com/users/caryoscelus/events{/privacy}","received_events_url":"https://api.github.com/users/caryoscelus/received_events","type":"User","site_admin":false},"labels":[],"state":"open","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":7,"created_at":"2016-12-24T18:25:33Z","updated_at":"2017-01-04T19:47:00Z","closed_at":null,"author_association":"CONTRIBUTOR","pull_request":{"url":"https://api.github.com/repos/renpy/renpy/pulls/1100","html_url":"https://github.com/renpy/renpy/pull/1100","diff_url":"https://github.com/renpy/renpy/pull/1100.diff","patch_url":"https://github.com/renpy/renpy/pull/1100.patch"},"body":"This is work in progress.\r\n\r\nI try to keep compatibility with py2 as well for now.","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1100/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1099","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1099/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1099/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1099/events","html_url":"https://github.com/renpy/renpy/issues/1099","id":197466619,"number":1099,"title":"Add possiblity to view current stack via Developer Menu","user":{"login":"alexei-osipov","id":8040900,"avatar_url":"https://avatars1.githubusercontent.com/u/8040900?v=4","gravatar_id":"","url":"https://api.github.com/users/alexei-osipov","html_url":"https://github.com/alexei-osipov","followers_url":"https://api.github.com/users/alexei-osipov/followers","following_url":"https://api.github.com/users/alexei-osipov/following{/other_user}","gists_url":"https://api.github.com/users/alexei-osipov/gists{/gist_id}","starred_url":"https://api.github.com/users/alexei-osipov/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/alexei-osipov/subscriptions","organizations_url":"https://api.github.com/users/alexei-osipov/orgs","repos_url":"https://api.github.com/users/alexei-osipov/repos","events_url":"https://api.github.com/users/alexei-osipov/events{/privacy}","received_events_url":"https://api.github.com/users/alexei-osipov/received_events","type":"User","site_admin":false},"labels":[],"state":"open","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":0,"created_at":"2016-12-24T15:03:34Z","updated_at":"2016-12-24T15:03:34Z","closed_at":null,"author_association":"NONE","body":"Related to #464.\r\nIt would be nice to have a possibility to view game stack via Developer Menu.\r\n\r\n","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1099/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1098","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1098/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1098/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1098/events","html_url":"https://github.com/renpy/renpy/issues/1098","id":197460075,"number":1098,"title":"Python 3 porting","user":{"login":"caryoscelus","id":1758893,"avatar_url":"https://avatars3.githubusercontent.com/u/1758893?v=4","gravatar_id":"","url":"https://api.github.com/users/caryoscelus","html_url":"https://github.com/caryoscelus","followers_url":"https://api.github.com/users/caryoscelus/followers","following_url":"https://api.github.com/users/caryoscelus/following{/other_user}","gists_url":"https://api.github.com/users/caryoscelus/gists{/gist_id}","starred_url":"https://api.github.com/users/caryoscelus/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/caryoscelus/subscriptions","organizations_url":"https://api.github.com/users/caryoscelus/orgs","repos_url":"https://api.github.com/users/caryoscelus/repos","events_url":"https://api.github.com/users/caryoscelus/events{/privacy}","received_events_url":"https://api.github.com/users/caryoscelus/received_events","type":"User","site_admin":false},"labels":[],"state":"open","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":3,"created_at":"2016-12-24T11:49:14Z","updated_at":"2017-07-22T06:17:17Z","closed_at":null,"author_association":"CONTRIBUTOR","body":"Is anybody currently working on this? I'll try it if nobody else does.\r\n\r\nDo you plan to switch quickly or with a transition period? I.e. should the porting effort be compatible with both versions (e.g. using six) or only py3?","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1098/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1097","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1097/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1097/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1097/events","html_url":"https://github.com/renpy/renpy/issues/1097","id":197105397,"number":1097,"title":"Setting config.quit_action to ui.gamemenus always open up the save screen","user":{"login":"DragoonHP","id":1311129,"avatar_url":"https://avatars3.githubusercontent.com/u/1311129?v=4","gravatar_id":"","url":"https://api.github.com/users/DragoonHP","html_url":"https://github.com/DragoonHP","followers_url":"https://api.github.com/users/DragoonHP/followers","following_url":"https://api.github.com/users/DragoonHP/following{/other_user}","gists_url":"https://api.github.com/users/DragoonHP/gists{/gist_id}","starred_url":"https://api.github.com/users/DragoonHP/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/DragoonHP/subscriptions","organizations_url":"https://api.github.com/users/DragoonHP/orgs","repos_url":"https://api.github.com/users/DragoonHP/repos","events_url":"https://api.github.com/users/DragoonHP/events{/privacy}","received_events_url":"https://api.github.com/users/DragoonHP/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2016-12-22T06:39:50Z","updated_at":"2016-12-22T22:07:28Z","closed_at":"2016-12-22T22:05:38Z","author_association":"CONTRIBUTOR","body":"No matter what value is provided for ui.gamemenus, it will always open the save screen\r\n\r\n```\r\nconfig.quit_action = ui.gamemenus(\"_quit_prompt\")\r\nconfig.quit_action = ui.gamemenus(\"_quit_prompt_dummy_name\")\r\nconfig.quit_action = ui.gamemenus(\"_main_menu_prompt\")\r\n```\r\nall open the save screen,\r\n(It worked perfectly in v6.99.11.1749)","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1097/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1096","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1096/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1096/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1096/events","html_url":"https://github.com/renpy/renpy/issues/1096","id":197104255,"number":1096,"title":"Launcher crashes when building a Mac package","user":{"login":"DragoonHP","id":1311129,"avatar_url":"https://avatars3.githubusercontent.com/u/1311129?v=4","gravatar_id":"","url":"https://api.github.com/users/DragoonHP","html_url":"https://github.com/DragoonHP","followers_url":"https://api.github.com/users/DragoonHP/followers","following_url":"https://api.github.com/users/DragoonHP/following{/other_user}","gists_url":"https://api.github.com/users/DragoonHP/gists{/gist_id}","starred_url":"https://api.github.com/users/DragoonHP/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/DragoonHP/subscriptions","organizations_url":"https://api.github.com/users/DragoonHP/orgs","repos_url":"https://api.github.com/users/DragoonHP/repos","events_url":"https://api.github.com/users/DragoonHP/events{/privacy}","received_events_url":"https://api.github.com/users/DragoonHP/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":0,"created_at":"2016-12-22T06:30:45Z","updated_at":"2016-12-22T22:30:21Z","closed_at":"2016-12-22T22:30:21Z","author_association":"CONTRIBUTOR","body":"\r\nA few points:\r\n * A max build is created;\r\n * Build Updates option is unchecked;\r\n * This was a project created in earlier version of Ren'Py.\r\n\r\nThis is the error message\r\n\r\n```\r\nI'm sorry, but an uncaught exception occurred.\r\n\r\nWhile running game code:\r\n  File \"game/distribute.rpy\", line 1462, in script\r\n  File \"game/distribute.rpy\", line 1469, in <module>\r\n  File \"game/distribute.rpy\", line 551, in __init__\r\n  File \"game/distribute.rpy\", line 1289, in make_package\r\nWindowsError: [Error 2] The system cannot find the file specified: u\"G:\\\\Ren'Py\\\\Projects\\\\Founder-Life---Recode-v0.1.1.1222-dists\\\\Founder-Life---Recode-v0.1.1.1222-mac.update.json\"\r\n\r\n-- Full Traceback ------------------------------------------------------------\r\n\r\nFull traceback:\r\n  File \"game/distribute.rpy\", line 1462, in script\r\n  File \"G:\\Ren'Py\\renpy\\ast.py\", line 814, in execute\r\n    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)\r\n  File \"G:\\Ren'Py\\renpy\\python.py\", line 1695, in py_exec_bytecode\r\n    exec bytecode in globals, locals\r\n  File \"game/distribute.rpy\", line 1469, in <module>\r\n  File \"game/distribute.rpy\", line 551, in __init__\r\n  File \"game/distribute.rpy\", line 1289, in make_package\r\nWindowsError: [Error 2] The system cannot find the file specified: u\"G:\\\\Ren'Py\\\\Projects\\\\Founder-Life---Recode-v0.1.1.1222-dists\\\\Founder-Life---Recode-v0.1.1.1222-mac.update.json\"\r\n\r\nWindows-8-6.2.9200\r\nRen'Py 6.99.12.2003\r\nRen'Py Launcher 6.99.12.2003\r\n\r\n```","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1096/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1095","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1095/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1095/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1095/events","html_url":"https://github.com/renpy/renpy/issues/1095","id":197032472,"number":1095,"title":"MultiPersistent broken on 6.99.12","user":{"login":"Ria-kon","id":24704640,"avatar_url":"https://avatars2.githubusercontent.com/u/24704640?v=4","gravatar_id":"","url":"https://api.github.com/users/Ria-kon","html_url":"https://github.com/Ria-kon","followers_url":"https://api.github.com/users/Ria-kon/followers","following_url":"https://api.github.com/users/Ria-kon/following{/other_user}","gists_url":"https://api.github.com/users/Ria-kon/gists{/gist_id}","starred_url":"https://api.github.com/users/Ria-kon/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/Ria-kon/subscriptions","organizations_url":"https://api.github.com/users/Ria-kon/orgs","repos_url":"https://api.github.com/users/Ria-kon/repos","events_url":"https://api.github.com/users/Ria-kon/events{/privacy}","received_events_url":"https://api.github.com/users/Ria-kon/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":2,"created_at":"2016-12-21T21:05:01Z","updated_at":"2016-12-27T00:02:13Z","closed_at":"2016-12-27T00:02:13Z","author_association":"CONTRIBUTOR","body":"I just reinstall RenPy to 6.99.12 from 6.99.11 and than i take only one issue with MP. Everything work on older version, but with newest version MP just seems like something not working. \r\nAfter playing with reencoding utf-8 script file and even separating code i just create new one project and paste renpy example in new file:\r\n```\r\ninit python:\r\n    mp = MultiPersistent(\"demo.renpy.org\")\r\n```\r\nAnd hooray, even this not working!\r\n```\r\nWhile running game code:\r\n  File \"game/1.rpy\", line 2, in <module>\r\n    mp = MultiPersistent(\"demo.renpy.org\")\r\nUnicodeDecodeError: 'ascii' codec can't decode byte 0xc0 in position 9: ordinal not in range(128)\r\n\r\n-- Full Traceback ------------------------------------------------------------\r\n\r\nFull traceback:\r\n  File \"lib/windows-i686/1.rpyc\", line 1, in script\r\n  File \"C:\\renpy-6.99.12-sdk\\renpy\\ast.py\", line 814, in execute\r\n    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)\r\n  File \"C:\\renpy-6.99.12-sdk\\renpy\\python.py\", line 1695, in py_exec_bytecode\r\n    exec bytecode in globals, locals\r\n  File \"game/1.rpy\", line 2, in <module>\r\n    mp = MultiPersistent(\"demo.renpy.org\")\r\n  File \"C:\\renpy-6.99.12-sdk\\renpy\\persistent.py\", line 469, in MultiPersistent\r\n    fn = fn + \"/\" + name\r\nUnicodeDecodeError: 'ascii' codec can't decode byte 0xc0 in position 9: ordinal not in range(128)\r\n```\r\nAnd i attached test project with log, traceback and 1.rpy (just with multipersistent function) and everything else\r\n[1.zip](https://github.com/renpy/renpy/files/667334/1.zip)","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1095/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1094","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1094/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1094/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1094/events","html_url":"https://github.com/renpy/renpy/pull/1094","id":195899015,"number":1094,"title":"Update spanish translation","user":{"login":"LuqueDaniel","id":1286535,"avatar_url":"https://avatars0.githubusercontent.com/u/1286535?v=4","gravatar_id":"","url":"https://api.github.com/users/LuqueDaniel","html_url":"https://github.com/LuqueDaniel","followers_url":"https://api.github.com/users/LuqueDaniel/followers","following_url":"https://api.github.com/users/LuqueDaniel/following{/other_user}","gists_url":"https://api.github.com/users/LuqueDaniel/gists{/gist_id}","starred_url":"https://api.github.com/users/LuqueDaniel/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/LuqueDaniel/subscriptions","organizations_url":"https://api.github.com/users/LuqueDaniel/orgs","repos_url":"https://api.github.com/users/LuqueDaniel/repos","events_url":"https://api.github.com/users/LuqueDaniel/events{/privacy}","received_events_url":"https://api.github.com/users/LuqueDaniel/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":0,"created_at":"2016-12-15T19:48:11Z","updated_at":"2016-12-16T06:00:41Z","closed_at":"2016-12-16T06:00:41Z","author_association":"MEMBER","pull_request":{"url":"https://api.github.com/repos/renpy/renpy/pulls/1094","html_url":"https://github.com/renpy/renpy/pull/1094","diff_url":"https://github.com/renpy/renpy/pull/1094.diff","patch_url":"https://github.com/renpy/renpy/pull/1094.patch"},"body":"Update spanish translation.","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1094/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1093","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1093/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1093/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1093/events","html_url":"https://github.com/renpy/renpy/issues/1093","id":195236226,"number":1093,"title":"Font error on translated new gui game","user":{"login":"nyaatrap","id":3129985,"avatar_url":"https://avatars3.githubusercontent.com/u/3129985?v=4","gravatar_id":"","url":"https://api.github.com/users/nyaatrap","html_url":"https://github.com/nyaatrap","followers_url":"https://api.github.com/users/nyaatrap/followers","following_url":"https://api.github.com/users/nyaatrap/following{/other_user}","gists_url":"https://api.github.com/users/nyaatrap/gists{/gist_id}","starred_url":"https://api.github.com/users/nyaatrap/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/nyaatrap/subscriptions","organizations_url":"https://api.github.com/users/nyaatrap/orgs","repos_url":"https://api.github.com/users/nyaatrap/repos","events_url":"https://api.github.com/users/nyaatrap/events{/privacy}","received_events_url":"https://api.github.com/users/nyaatrap/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":0,"created_at":"2016-12-13T12:07:07Z","updated_at":"2016-12-16T05:22:30Z","closed_at":"2016-12-16T05:22:30Z","author_association":"CONTRIBUTOR","body":"Found what causes an error we were talked before.\r\n\r\n![clipboard 1](https://cloud.githubusercontent.com/assets/3129985/21139780/ef3eae48-c177-11e6-92b5-05fc0a956359.png)\r\n[traceback.txt](https://github.com/renpy/renpy/files/648874/traceback.txt)\r\n","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1093/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1092","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1092/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1092/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1092/events","html_url":"https://github.com/renpy/renpy/issues/1092","id":195224221,"number":1092,"title":"Text outlines don't scale along with text when game window is resized","user":{"login":"dansalvato","id":7810794,"avatar_url":"https://avatars1.githubusercontent.com/u/7810794?v=4","gravatar_id":"","url":"https://api.github.com/users/dansalvato","html_url":"https://github.com/dansalvato","followers_url":"https://api.github.com/users/dansalvato/followers","following_url":"https://api.github.com/users/dansalvato/following{/other_user}","gists_url":"https://api.github.com/users/dansalvato/gists{/gist_id}","starred_url":"https://api.github.com/users/dansalvato/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/dansalvato/subscriptions","organizations_url":"https://api.github.com/users/dansalvato/orgs","repos_url":"https://api.github.com/users/dansalvato/repos","events_url":"https://api.github.com/users/dansalvato/events{/privacy}","received_events_url":"https://api.github.com/users/dansalvato/received_events","type":"User","site_admin":false},"labels":[],"state":"open","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":0,"created_at":"2016-12-13T11:09:08Z","updated_at":"2016-12-13T11:09:08Z","closed_at":null,"author_association":"NONE","body":"If a text style has outlines, then the thickness of the outlines doesn't increase along with everything else being scaled up. For example, my game window is 1280x720, but when I enter fullscreen, my screen size is 1920x1080. This causes outlines to look much thinner when in fullscreen.\r\n\r\nTo help demonstrate this, I'll use a more extreme example of resolution differences.\r\n\r\nText outline at 640x360:\r\n![image](https://cloud.githubusercontent.com/assets/7810794/21138070/acf1f678-c0e9-11e6-92b4-d7a52c8de56b.png)\r\n\r\nThe same text at 1920x1080:\r\n![image](https://cloud.githubusercontent.com/assets/7810794/21138077/b5b4e6bc-c0e9-11e6-885b-3180d0af875f.png)\r\n\r\nSince the text was scaled up, but the outline remains the same size, the outline appears much thinner.","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1092/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1091","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1091/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1091/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1091/events","html_url":"https://github.com/renpy/renpy/issues/1091","id":194841858,"number":1091,"title":"Request: im.matrix.brightnesscutoff(alpha,invert=False)","user":{"login":"Kuroonehalf","id":12227163,"avatar_url":"https://avatars1.githubusercontent.com/u/12227163?v=4","gravatar_id":"","url":"https://api.github.com/users/Kuroonehalf","html_url":"https://github.com/Kuroonehalf","followers_url":"https://api.github.com/users/Kuroonehalf/followers","following_url":"https://api.github.com/users/Kuroonehalf/following{/other_user}","gists_url":"https://api.github.com/users/Kuroonehalf/gists{/gist_id}","starred_url":"https://api.github.com/users/Kuroonehalf/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/Kuroonehalf/subscriptions","organizations_url":"https://api.github.com/users/Kuroonehalf/orgs","repos_url":"https://api.github.com/users/Kuroonehalf/repos","events_url":"https://api.github.com/users/Kuroonehalf/events{/privacy}","received_events_url":"https://api.github.com/users/Kuroonehalf/received_events","type":"User","site_admin":false},"labels":[],"state":"open","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":0,"created_at":"2016-12-11T16:48:23Z","updated_at":"2016-12-11T16:51:00Z","closed_at":null,"author_association":"CONTRIBUTOR","body":"This function would receive a float argument 'alpha' between 0 and 1, and make any pixel with brightness bellow that number pure white, and any pixel above that pure black.\r\n\r\nInvert would switch the whites and blacks.\r\n\r\nHere's a practical example of the intended behavior.\r\nUsing this as our base image\r\n![gradient](https://cloud.githubusercontent.com/assets/12227163/21081547/42d58e0e-bfc1-11e6-9e05-2dde27fcee0c.jpg)\r\n \r\nIf we did `im.MatrixColor(\"mask.png\", im.matrix.brightnesscutoff(0.5))` we would get:\r\n![cutoff05](https://cloud.githubusercontent.com/assets/12227163/21081510/a826b93c-bfc0-11e6-9287-67e9c395b3e6.jpg)\r\n\r\nAnd `im.MatrixColor(\"mask.png\", im.matrix.brightnesscutoff(0.8))` we'd get:\r\n![cutoff08](https://cloud.githubusercontent.com/assets/12227163/21081541/ff23e5f2-bfc0-11e6-90fe-f280567b7b9a.jpg)\r\n\r\nThis, coupled with im.AlphaMask() should allow devs to do things such as curved health bars, or other progressive cutoff image elements that aren't straight, which is something I've personally been wanting for a while.","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1091/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1090","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1090/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1090/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1090/events","html_url":"https://github.com/renpy/renpy/pull/1090","id":194811772,"number":1090,"title":"Update Japanese translation","user":{"login":"nyaatrap","id":3129985,"avatar_url":"https://avatars3.githubusercontent.com/u/3129985?v=4","gravatar_id":"","url":"https://api.github.com/users/nyaatrap","html_url":"https://github.com/nyaatrap","followers_url":"https://api.github.com/users/nyaatrap/followers","following_url":"https://api.github.com/users/nyaatrap/following{/other_user}","gists_url":"https://api.github.com/users/nyaatrap/gists{/gist_id}","starred_url":"https://api.github.com/users/nyaatrap/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/nyaatrap/subscriptions","organizations_url":"https://api.github.com/users/nyaatrap/orgs","repos_url":"https://api.github.com/users/nyaatrap/repos","events_url":"https://api.github.com/users/nyaatrap/events{/privacy}","received_events_url":"https://api.github.com/users/nyaatrap/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":0,"created_at":"2016-12-11T04:19:42Z","updated_at":"2016-12-11T04:20:27Z","closed_at":"2016-12-11T04:20:27Z","author_association":"CONTRIBUTOR","pull_request":{"url":"https://api.github.com/repos/renpy/renpy/pulls/1090","html_url":"https://github.com/renpy/renpy/pull/1090","diff_url":"https://github.com/renpy/renpy/pull/1090.diff","patch_url":"https://github.com/renpy/renpy/pull/1090.patch"},"body":"","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1090/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1089","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1089/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1089/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1089/events","html_url":"https://github.com/renpy/renpy/pull/1089","id":194326065,"number":1089,"title":"Remove trailing ' from comment for translation","user":{"login":"nyaatrap","id":3129985,"avatar_url":"https://avatars3.githubusercontent.com/u/3129985?v=4","gravatar_id":"","url":"https://api.github.com/users/nyaatrap","html_url":"https://github.com/nyaatrap","followers_url":"https://api.github.com/users/nyaatrap/followers","following_url":"https://api.github.com/users/nyaatrap/following{/other_user}","gists_url":"https://api.github.com/users/nyaatrap/gists{/gist_id}","starred_url":"https://api.github.com/users/nyaatrap/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/nyaatrap/subscriptions","organizations_url":"https://api.github.com/users/nyaatrap/orgs","repos_url":"https://api.github.com/users/nyaatrap/repos","events_url":"https://api.github.com/users/nyaatrap/events{/privacy}","received_events_url":"https://api.github.com/users/nyaatrap/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":0,"created_at":"2016-12-08T12:40:45Z","updated_at":"2016-12-08T15:12:56Z","closed_at":"2016-12-08T15:12:56Z","author_association":"CONTRIBUTOR","pull_request":{"url":"https://api.github.com/repos/renpy/renpy/pulls/1089","html_url":"https://github.com/renpy/renpy/pull/1089","diff_url":"https://github.com/renpy/renpy/pull/1089.diff","patch_url":"https://github.com/renpy/renpy/pull/1089.patch"},"body":"Now I think all translation issues are over.","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1089/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}},{"url":"https://api.github.com/repos/renpy/renpy/issues/1088","repository_url":"https://api.github.com/repos/renpy/renpy","labels_url":"https://api.github.com/repos/renpy/renpy/issues/1088/labels{/name}","comments_url":"https://api.github.com/repos/renpy/renpy/issues/1088/comments","events_url":"https://api.github.com/repos/renpy/renpy/issues/1088/events","html_url":"https://github.com/renpy/renpy/pull/1088","id":194230813,"number":1088,"title":"Update Japanese translation","user":{"login":"nyaatrap","id":3129985,"avatar_url":"https://avatars3.githubusercontent.com/u/3129985?v=4","gravatar_id":"","url":"https://api.github.com/users/nyaatrap","html_url":"https://github.com/nyaatrap","followers_url":"https://api.github.com/users/nyaatrap/followers","following_url":"https://api.github.com/users/nyaatrap/following{/other_user}","gists_url":"https://api.github.com/users/nyaatrap/gists{/gist_id}","starred_url":"https://api.github.com/users/nyaatrap/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/nyaatrap/subscriptions","organizations_url":"https://api.github.com/users/nyaatrap/orgs","repos_url":"https://api.github.com/users/nyaatrap/repos","events_url":"https://api.github.com/users/nyaatrap/events{/privacy}","received_events_url":"https://api.github.com/users/nyaatrap/received_events","type":"User","site_admin":false},"labels":[],"state":"closed","locked":false,"assignee":null,"assignees":[],"milestone":null,"comments":1,"created_at":"2016-12-08T02:25:20Z","updated_at":"2016-12-08T04:26:49Z","closed_at":"2016-12-08T04:26:41Z","author_association":"CONTRIBUTOR","pull_request":{"url":"https://api.github.com/repos/renpy/renpy/pulls/1088","html_url":"https://github.com/renpy/renpy/pull/1088","diff_url":"https://github.com/renpy/renpy/pull/1088.diff","patch_url":"https://github.com/renpy/renpy/pull/1088.patch"},"body":"Include weekday on fileslot.","reactions":{"url":"https://api.github.com/repos/renpy/renpy/issues/1088/reactions","total_count":0,"+1":0,"-1":0,"laugh":0,"hooray":0,"confused":0,"heart":0}}],"meta":{"x-ratelimit-limit":"5000","x-ratelimit-remaining":"4819","x-ratelimit-reset":"1507317096","x-github-request-id":"D9C2:366E:E5F41:1F8E8F:59D7D086","x-github-media-type":"github.squirrel-girl-preview","link":"<https://api.github.com/repositories/4815490/issues?state=all&page=7>; rel=\"next\", <https://api.github.com/repositories/4815490/issues?state=all&page=43>; rel=\"last\", <https://api.github.com/repositories/4815490/issues?state=all&page=1>; rel=\"first\", <https://api.github.com/repositories/4815490/issues?state=all&page=5>; rel=\"prev\"","etag":"\"263fb3102f2941efaff9e8947e343898\"","status":"200 OK"}}
